### PR TITLE
feature/add-filter-by-instructor|Added the filter by instructor for t…

### PR DESCRIPTION
**Pull request by oleh-rymar**
…he Repeat Schedules block.

### CODE REVIEW

File: templates/openy-repeat-schedule-dashboard--sidebar.html.twig [hunk 1]
```diff
@@ -138,6 +138,26 @@
             </div>
           </div>
         {% endif %}
+
+        {# inscructor name select list #}
```
Comment (froboy) [inline templates/openy-repeat-schedule-dashboard--sidebar.html.twig:142]:
```suggestion
        {# instructor name select list #}
```

File: modules/repeat_schedules_instructor/config/install/field.storage.paragraph.field_prgf_repeat_schedule_filt.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+settings:
+  allowed_values:
+    -
+      value: category
+      label: 'by category'
+    -
+      value: classname
+      label: 'by class name'
+    -
+      value: inscructorname
```
Comment (froboy) [inline modules/repeat_schedules_instructor/config/install/field.storage.paragraph.field_prgf_repeat_schedule_filt.yml:20]:
```suggestion
      value: instructorname
```

File: templates/openy-repeat-schedule-dashboard--sidebar.html.twig [hunk 3]
```diff
@@ -138,6 +138,26 @@
...
+        {% if filters.inscructorname %}
```
Comment (froboy) [inline templates/openy-repeat-schedule-dashboard--sidebar.html.twig:143]:
```suggestion
        {% if filters.instructorname %}
```

**Review by froboy — CHANGES_REQUESTED** [review]
@oleh-rymar this looks good. I left a few small comments. Could you please add test steps and some screenshots of the expected results when the module is enabled?

**Comment by oleh-rymar** [comment]
@froboy 
Fixed. Please, check.

**Review by froboy — APPROVED** [review]
Looks great. Thanks for the responses!

**Comment by podarok** [comment]
I think height of the element in Dropdown is not correct
We need to do better UX

image

**Comment by oleh-rymar** [comment]
Screenshot from 2024-04-25 10-16-46
@podarok @froboy 
Updated.

**Comment by podarok** [comment]
See https://yusaslackinstance.slack.com/archives/C03E6SQQ0Q5/p1714061856384559?thread_ts=1713557003.572269&cid=C03E6SQQ0Q5 @oleh-rymar 

I'm merging this as it is functional
BUt if we can do select list as dropdown menu from https://www.w3schools.com/css/css_dropdowns.asp - that would be great

**Pull request merged**